cli/cmd/outputs.go

```diff
@@ -64,3 +65,32 @@ func (c *cliState) FormatJSONString(s string) (string, error) {
 	}
 	return string(pretty), nil
 }
+
+// Used to clean CSV inputs prior to rendering
+func csvCleanData(input []string) []string {
+	var data []string
+	for _, h := range input {
+		data = append(data, strings.Replace(h, "\n", "", -1))
+	}
+	return data
+}
+
+// Used to produce CSV output
+func renderAsCSV(headers []string, data [][]string) string {
+	csvOut := &strings.Builder{}
+	csv := csv.NewWriter(csvOut)
+
+	if len(headers) > 0 {
+		csv.Write(csvCleanData(headers))
```

Comment:
The test build pipeline is failing here on linting check. Can you run a `make lint` and correct the issue its throwing.